public/stylesheets/site/2.0/21-userstuff.css

```diff
@@ -139,6 +139,11 @@
   border: 1px solid;
 }
 
+.userstuff img {
+  max-width: 100%;
+  height: auto;
```

Comment:
Is this to preserve images' aspect ratios?

Reply:
Yup! It'll override any `height` attribute the user set on the `img` tag to make sure nothing gets stretched out.